android/src/main/java/io/parity/signer/components/networkicon/NetworkIcon.kt

```diff
@@ -134,6 +134,8 @@ private fun getResourceIdForNetwork(networkName: String) =
 		"efinity" -> R.drawable.network_efinity
 		"encointer-parachain" -> R.drawable.network_encointer_parachain
 		"equilibrium" -> R.drawable.network_equilibrium
+		"frequency" -> R.drawable.network_frequency
+		"frequency-rococo" -> R.drawable.network_frequency_rococo
```

Comment:
```suggestion
		"frequency-rococo" -> R.drawable.network_frequency
```

@rlaferla android build is failing because this resource "network_frequency_rococo" you referenced is not added.
As I see it is the same icon for both networks - then I would suggest to reference the same resource - no need to add another copy for android.